breakdown-guides.rst

```diff
@@ -0,0 +1,402 @@
+
+Schedule Impact
+	Categorization of schedule impact, relative to the critical path of the Rubin Observatory's schedule (e.g., the data release cycle).
+
+	You should discuss the specifics with your department's Associate Director to determine the schedule impact.
+	For example, some delays may have an inconsequential impact to the Observatory's operations if it can be absorbed into the data release cycle, while others may require extending the LSST survey or delaying a data release.
+	The latter example affects the Observatory's operational critical path and crucial milestones --- these impacts are the most important ones to capture and accurately.
+
+	See :ref:`Risk-Impact-Table` for categories.
+
+Likelihood
+	Overall chance of risk being realized.
+
+	See :ref:`Likelihood-Table` for categories.
+
+Existential Risk
+	Select ``Yes`` if the risk is existential to NOIRLab.
+
+	The Rubin Observatory Risk and Opportunity Board will determine if this is appropriate.
+
+Schedule/Cost Impact Description
+	Use these sections to describe and comment on impact categorization and the decision on rating.
+
+
+.. _Breakdown-Risk-Score-Quantitative:
+
+Risk score and quantitative analysis 
+------------------------------------
+
+The fields under ``Risk Score`` are automatically generated based on input selections from :ref:`risk impacts <Breakdown-Risk-Initial-Impact>`.
+Categories and values are based on the :ref:`Breakdown-Risk-Tool-Tables`.
+The ``Analyze Risk Quantitative`` section will not affect values and categories; however, the section will record the impact justification and provide information needed to categorize the impacts.
+
+.. figure:: /_static/Risk-Example-Analyze-Risk-Score-and-Quantitative.png
+    :name: Risk-Example-Analyze-Risk-Score-and-Quantitative
+
+    Risk Score and Analysis Quantitative sections using an example risk.
+
+Impact Severity
+	Automatically generated category based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.
+
+	See :ref:`Risk-Impact-Table` for categories.
+
+Impact Score
+	Automatically generated value based on ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.
+
+Likelihood Score
+	Automatically generated value based on ``Likelihood``.
+
+Probability
+	Automatically generated value based on ``Likelihood``.
+
+Initial Risk Score
+	Automatically generated value based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.
+
+Minimum Delay (Months)
+	Minimum delay if risk is realized.
+
+Maximum Delay (Months)
+	Maximum delay if risk is realized.
+
+Likely Delay (Months)
+	Most likely delay if risk is realized.
+
+Expected Schedule Delay (Months)
+	Automatically generated value based on ``Minimum Delay``, ``Maximum Delay``, and ``Likely Delay``.
+
+.. todo::
+   Define formula for Expected Schedule Delay.
+
+Impact Time
+	Date when realized risk would impact the schedule.
+
+Impacted Event/Milestone
+	The event or milestone of a realized risk.
```

Comment:
```suggestion
	The event or milestone of a realized risk. This is important, so that the meaning of the schedule delay is clear. Choose from: LSST Survey Start, DR1, DR2, Year 1 Annual Maintenance, ... DR11, LSST Survey Finish.
```

Reply:
Added to next commit. Because you seem to indicate a full list of milestones to choose from, I've included a todo item to list out the milestones.